Traceback (most recent call last):
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\transformers\utils\import_utils.py", line 1093, in _get_module
    return importlib.import_module("." + module_name, self.__name__)
  File "C:\Python310\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\transformers\trainer.py", line 67, in <module>
    from .data.data_collator import DataCollator, DataCollatorWithPadding, default_data_collator
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\transformers\data\__init__.py", line 30, in <module>
    from .metrics import glue_compute_metrics, xnli_compute_metrics
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\transformers\data\metrics\__init__.py", line 22, in <module>
    if is_sklearn_available():
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\transformers\utils\import_utils.py", line 541, in is_sklearn_available
    return is_scipy_available() and importlib.util.find_spec("sklearn.metrics")
  File "C:\Python310\lib\importlib\util.py", line 94, in find_spec
    parent = __import__(parent_name, fromlist=['__path__'])
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\sklearn\__init__.py", line 82, in <module>
    from .base import clone
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\sklearn\base.py", line 17, in <module>
    from .utils import _IS_32BIT
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\sklearn\utils\__init__.py", line 21, in <module>
    from . import _joblib
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\sklearn\utils\_joblib.py", line 7, in <module>
    import joblib
ModuleNotFoundError: No module named 'joblib'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\omnic\local\GitRepos\chat-berta\pkg-dependencies\model_test.py", line 37, in test_model_usage
    from apiutils.nec.NetworkComponent import NetworkComponent
  File "C:\Users\omnic\local\GitRepos\chat-berta\apiutils\nec\NetworkComponent.py", line 1, in <module>
    from apiutils.model.ChatBerta import ChatBerta
  File "C:\Users\omnic\local\GitRepos\chat-berta\apiutils\model\ChatBerta.py", line 4, in <module>
    from apiutils.model.chat_bert_trainer import CustomTrainer
  File "C:\Users\omnic\local\GitRepos\chat-berta\apiutils\model\chat_bert_trainer.py", line 2, in <module>
    from transformers import Trainer
  File "<frozen importlib._bootstrap>", line 1075, in _handle_fromlist
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\transformers\utils\import_utils.py", line 1083, in __getattr__
    module = self._get_module(self._class_to_module[name])
  File "C:\Users\omnic\local\GitRepos\chat-berta\venv\lib\site-packages\transformers\utils\import_utils.py", line 1095, in _get_module
    raise RuntimeError(
RuntimeError: Failed to import transformers.trainer because of the following error (look up to see its traceback):
No module named 'joblib'
